src/server/handlers/dynamic.js

```diff
@@ -32,6 +35,17 @@ export default ({ server, config }) => {
     method: 'GET',
     path: '/{path*}',
     handler: async function(request, h) {
+      // Set a cache key
+      const cacheKey = request.url.pathname
```

Comment:
Do we need to normalise this? Or will react-router-config help us out here?

Reply:
Added `/` fallback for the homepage. Otherwise I think it's alright?